SmileMath/src/main/java/smile/math/kernel/HellingerKernel.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Diego Catalano
+ *   
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *  
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package smile.math.kernel;
+
+import smile.math.Math;
+
+/**
+ * The Hellinger Mercer Kernel.
```

Comment:
Can you add some comments about this kernel? The math formula, and suitable for what use cases. Is this for computer vision? Thanks!